extensions/database/test/conf/int_tests.xml

```diff
@@ -1,41 +1,56 @@
+        to the database to test. All tests which rely on
+        this database will then run.
+    -->
+
+    <parameter name = "mySqlDbName" value="test_db"/> 
+    <parameter name = "mySqlDbHost" value="127.0.0.1"/> 
+    <parameter name = "mySqlDbPort" value="3306"/> 
+    <parameter name = "mySqlDbUser" value="root"/>
     <parameter name = "mySqlDbPassword" value=""/>
     <parameter name = "mySqlTestTable" value="test_table"/>
     
-	<parameter name = "pgSqlDbName" value="test_db"/> 
-	<parameter name = "pgSqlDbHost" value="127.0.0.1"/> 
-	<parameter name = "pgSqlDbPort" value="5432"/> 
-	<parameter name = "pgSqlDbUser" value="postgres"/>
+    <parameter name = "pgSqlDbName" value="test_db"/> 
+    <parameter name = "pgSqlDbHost" value="127.0.0.1"/> 
+    <parameter name = "pgSqlDbPort" value="5432"/> 
+    <parameter name = "pgSqlDbUser" value="postgres"/>
     <parameter name = "pgSqlDbPassword" value=""/>
     <parameter name = "pgSqlTestTable" value="test_table"/>
     
     <parameter name = "mariaDbName" value="test_db"/> 
-	<parameter name = "mariaDbHost" value="127.0.0.1"/> 
-	<parameter name = "mariadbDbPort" value="3306"/> 
-	<parameter name = "mariaDbUser" value="root"/>
+    <parameter name = "mariaDbHost" value="127.0.0.1"/> 
+    <parameter name = "mariadbDbPort" value="3306"/> 
+    <parameter name = "mariaDbUser" value="root"/>
     <parameter name = "mariaDbPassword" value=""/>
     <parameter name = "mariaDbTestTable" value="test_table"/>
-	<test name="DatabaseExtensionUnitTest">
-		<classes>
-			<class name="com.google.refine.extension.database.DatabaseImportControllerTest"/>
-			<class name="com.google.refine.extension.database.DatabaseServiceTest"/>
-			<class name="com.google.refine.extension.database.SimpleTextEncryptorTest"/>
-			<class name="com.google.refine.extension.database.cmd.ConnectCommandTest"/>
-			<class name="com.google.refine.extension.database.cmd.ExecuteQueryCommandTest"/>
-			<class name="com.google.refine.extension.database.cmd.SavedConnectionCommandTest"/>
-			<class name="com.google.refine.extension.database.cmd.TestConnectCommandTest"/>
-			<class name="com.google.refine.extension.database.cmd.TestQueryCommandTest"/>
-			<class name="com.google.refine.extension.database.mariadb.MariaDBConnectionManagerTest"/>
-			<class name="com.google.refine.extension.database.mariadb.MariaDBDatabaseServiceTest"/>
-		 	<class name="com.google.refine.extension.database.mysql.MySQLConnectionManagerTest"/>
-			<class name="com.google.refine.extension.database.mysql.MySQLDatabaseServiceTest"/>
-		    <class name="com.google.refine.extension.database.pgsql.PgSQLConnectionManagerTest"/>
-			<class name="com.google.refine.extension.database.pgsql.PgSQLDatabaseServiceTest" />
-		</classes>
-	</test>
-</suite>
\ No newline at end of file
+    <test name="DatabaseExtensionUnitTest">
+        <groups>
+            <run>
+                <exclude name="requiresMySQL"/>
```

Comment:
why exclude all db-specific testing? only do it on travis?

Reply:
I am not sure what you mean - we already have three files for local, travis and appveyor (the fourth one should be deleted - I am just waiting for @tcbuzor's confirmation). It's not clear to me how to avoid the copy but feel free to commit on this branch with the setup you are thinking about. Note that currently no copy is needed when you test locally - the default configuration file already excludes all tests which require database access. Copying is only required on Travis and Appveyor (but that does not seem to be a big issue to me).